src/components/layout/GenericLayout/variablesCss.ts

```diff
@@ -1,9 +1,12 @@
 import { css } from 'styled-components'
 
 const AllColors = `
+  /* HEIGHTS */
+  --height-bar-default: 6.2rem;
+  /* ------------------------------ */
+
   /* FONTS */
   --font-default: "Inter", "Helvetica Neue", Helvetica, sans-serif;
-  --font-mono: "Roboto Mono", monospace;
```

Comment:
good call to do all this cleanup. We can leave it in this PR. But this one is better of in it's own independent PR
The smaller the PR, the better